wiki/sensing/robotic-total-stations.md

```diff
@@ -0,0 +1,88 @@
+---
+# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
+# overwritten except in special circumstances. 
+# You should set the date the article was last updated like this:
+date: 2022-12-05 # YYYY-MM-DD
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: Robotic Total Station (Leica TS16)
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+## Introduction
```

Comment:
The first paragraph (or the first 100 words) becomes the page description. The blurb for this page will just read "## Introduction"...

I'd recommend dropping this line and keeping the first page as-is. 

CC: @nevalsar